core/src/main/java/me/leoko/advancedban/MethodInterface.java

```diff
@@ -68,8 +68,13 @@
      */
     void setupMetrics();
 
+    @Deprecated
     boolean isBungee();
 
+    boolean isProxy();
+
+    String getProxyType();
```

Comment:
This is a good idea to add getProxyType(), but I'm wondering if it would make sense to use `getServerType()` instead. Then, BukkitMethods would return "Spigot" and the logic in the /advancedban would be simplified.

Perhaps even better than that would be a `ServerType` enum so code which uses getProxyType/getServerType isn't stringly-typed. That would replace `Universal.get().getProxyType().equals("Bungeecord")` with `Universal.get().getServerType() == ServerType.BUNGEECORD`

Reply:
Yeah - I was thinking about this when I wrote this but ran out of time at the time. I feel like it could be useful for this